tests/landIce/FO_AIS/inputMueLuKokkos.yaml

```diff
@@ -0,0 +1,342 @@
+%YAML 1.1
+---
+ANONYMOUS:
+  Build Type: Tpetra
+  Debug Output: {}
+  Problem:
+    Phalanx Graph Visualization Detail: 0
+    Solution Method: Steady
+    Name: LandIce Stokes First Order 3D
+    LandIce Rigid Body Modes For Preconditioner:
+      Compute Constant Modes: true
+      Compute Rotation Modes: true
+    Basal Side Name: basalside
+    Cubature Degree: 3
+    Basal Cubature Degree: 3
+    Flat Bed Approximation: true
+    Compute Sensitivities: true
+    Response Functions:
+      Number Of Responses: 1
+      Response 0:
+        Type: Scalar Response
+        Name: Solution Average
+    Dirichlet BCs: {}
+    Neumann BCs: {}
+    LandIce BCs:
+      Number: 2
+      BC 0:
+        Type: Basal Friction
+        Side Set Name: basalside
+        Basal Friction Coefficient:
+          Type: Field
+          Beta Field Name: basal_friction
+      BC 1:
+        Type: Lateral
+        Cubature Degree: 3
+        Side Set Name: lateralside
+    Parameters:
+      Number Of Parameters: 1
+      Parameter 0:
+        Type: Scalar
+        Name: Glen's Law Homotopy Parameter
+    LandIce Physical Parameters:
```

Comment:
Max, would you mind fixing the format of the floating point parameters in this input file? I'm trying to get rid of things like `9.99999999999999954e-08` in favor of `1e-07` in all input files.